model/utility/parser.php

```diff
@@ -2020,7 +2020,10 @@ public function setBreakOnError($break = true) {
 * @param $ednDescrip string describing the edition
 * @return array of parser configuration metadata
 */
-function createParserConfig($ownerID = 4,$vis,$attr,$ckn,$trid,$ktxtid,$textid,$mask,$order,$scribe,$translit,$side = null,$part = null,$fragment = null,$ednDescrip = null) {
```

Comment:
PHP8 considers ownerID an optional parameter when assigned in-line, and so objects to it preceding required parameters. Callers specify null for this position when null, so safe to assign to default value 4 inside the function.